File: engine/engine-model/src/main/scala/net/server/channel/packet/cash/use/UseOwlSearchPacket.scala
package net.server.channel.packet.cash.use

class UseOwlSearchPacket(private var _position: Short, private var _itemId: Int, private var _searchedItemId:Int) extends AbstractUseCashItemPacket(_position, _itemId) {
  def searchedItemId: Int = _searchedItemId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/use/UseOwlSearchPacket.java
package net.server.channel.packet.cash.use;

public class UseOwlSearchPacket extends AbstractUseCashItemPacket {
   private final Integer searchedItemId;

   public UseOwlSearchPacket(Short position, Integer itemId, Integer searchedItemId) {
      super(position, itemId);
      this.searchedItemId = searchedItemId;
   }

   public Integer searchedItemId() {
      return searchedItemId;
   }
}
